crawl-ref/source/dat/des/arrival/simple.des

```diff
@@ -2508,3 +2508,44 @@ tt.........tt
  tt.......tt
   ttt@@@ttt
 ENDMAP
+
+NAME:   reono_arrival_forest_river
+TAGS:   arrival no_monster_gen no_trap_gen no_pool_fixup
+ORIENT: float
+SUBST:  w = w:5 W:1 t:2
```

Comment:
This subst can create pockets of shallow water that aren't accessible without flight or, rarely, inaccessible disconnected areas behind trees. You could try substituting only the glyphs next to the outer walls to make the vault feel less boxy without generating disconnects.